src/main/kotlin/org/jetbrains/intellij/utils.kt

```diff
@@ -201,7 +202,8 @@ fun releaseType(version: String) = when {
     version.endsWith("-EAP-SNAPSHOT") ||
         version.endsWith("-EAP-CANDIDATE-SNAPSHOT") ||
         version.endsWith("-CUSTOM-SNAPSHOT") ||
-        version.matches(MAJOR_VERSION_PATTERN.toRegex())
+        version.matches(MAJOR_VERSION_PATTERN.toRegex()) ||
+        BUILD_VERSION_PATTERN.matcher(version).matches()
```

Comment:
this one is not good. I think it breaks nightly IDEA builds, that look like `201.1463-SNAPSHOT` and must be downloaded from `nightly` repo. 
GW should be either uploaded to `nightly` (access from JB network only), or to be renamed, e.g. with `-CUSTOM-SNAPSHOT` prefix (I cannot remember why we introduced it, though).